docker/task-server/Dockerfile

```diff
@@ -1,45 +1,15 @@
-FROM python:2.7
+#Inherit our django image which has the same pip requirements start the celery container.
+FROM freelawproject/courtlistener-django:bte_beta
 
 WORKDIR /usr/src/app
 
-# Use lots of layers. Bigger image, but faster builds. Slow stuff is up here at
-# the top.
-RUN apt-get update --option "Acquire::Retries=3" --quiet=2 && \
-    apt-get install -y --no-install-recommends apt-utils && \
-    apt-get install \
-        --option "Acquire::Retries=3" \
-        --no-install-recommends \
-        --assume-yes \
-        --quiet=2 \
-        `# Document extraction and OCR tools` \
-        antiword docx2txt ghostscript libwpd-tools poppler-utils \
-        `# Audio extraction/manipulation tools` \
-        ffmpeg \
-        `# Image & OCR tools` \
-        imagemagick tesseract-ocr tesseract-ocr-eng \
-        `# Other dependencies` \
-        libffi-dev libxml2-dev libxslt-dev python-dev
-
-COPY requirements.txt requirements-dev.txt ./
-RUN pip install --no-cache-dir -r requirements-dev.txt && \
-    pip install --no-cache-dir -r requirements.txt
-RUN pip install git+https://github.com/freelawproject/judge-pics@1.1.2 && \
-    update-seals -f
-
-COPY requirements-flp.txt ./
-RUN pip install --no-cache-dir -r requirements-flp.txt
-
-
 ## Needs to be two commands so second one can use variables from first.
 ENV \
     CELERY_TASKS_DIR=/opt/celery \
     CELERY_USER_ID=33
 ENV \
     PYTHONPATH="${PYTHONPATH}:${CELERY_TASKS_DIR}"
 
-RUN mkdir /var/log/courtlistener && \
-    chmod 777 /var/log/courtlistener
-
```

Comment:
You're sure we can ditch this?

Reply:
If my understanding of docker is right- its inherited line for line from the Django image

Reply:
Yes, OK, that does make sense. Awesome.